apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/persistence/RestApiV2QMaterializedViewIT.java

```diff
@@ -81,6 +81,14 @@ public void getRowsFromMV() {
     assertThat(resultSet.wasApplied()).isTrue();
 
     // And then read entries using MV:
+
+    // 14-Sep-2022, tatu: Not sure why but there is a transient issue here wrt timing.
```

Comment:
Non-elegant, but unfortunately attempts at using `@RetryableTest` were unsuccessful (does not work well together with "assumptions" since those are implemented via `AssertionException`s and taken as actual fails).